packages/discovery-provider/plugins/pedalboard/apps/solana-relay/src/config.ts

```diff
@@ -117,7 +120,10 @@ const readConfig = (): Config => {
     audius_solana_signer_private_key: str({
       default:
         'd242765e718801781440d77572b9dafcdc9baadf0269eff24cf61510ddbf1003'
-    })
+    }),
+    audius_solana_listens_hourly_rate_limit: num({ default: 10 }),
+    audius_solana_listens_daily_rate_limit: num({ default: 40 }),
+    audius_solana_listens_weekly_rate_limit: num({ default: 120 })
```

Comment:
from identity https://github.com/AudiusProject/audius-protocol/blob/dfa3b62e33c4e934346f71d81b40b2573340989f/packages/identity-service/default-config.json#L34-L40

Reply:
changed to IP because that's what this is checking

Reply:
i missed these defaults in identity yesterday 👀

Reply:
but the defaults are different for this one identity. they're much larger. the smaller ones are per ip+track

Reply:
cool cool